model-base.coffee

```diff
@@ -213,11 +213,11 @@ class AlienModelBase extends AlienCommander
       , options
     @
 
-  @belongsTo: (rel_name, model_name) ->
-    @_initRelation 'belongsTo', rel_name, model_name
+  @belongsTo: (rel_name, model_name, foreign_key) ->
+    @_initRelation 'belongsTo', rel_name, model_name, foreign_key: foreign_key
 
-  @hasMany: (rel_name, model_name) ->
-    @_initRelation 'hasMany', rel_name, model_name
+  @hasMany: (rel_name, model_name, foreign_key) ->
+    @_initRelation 'hasMany', rel_name, model_name, foreign_key: foreign_key
```

Comment:
Csak közepesen tetszik, h így az optionsban lesz foreign_key, csak undefined lesz, akkor is ha nincs.